docs/advanced_usage.md

```diff
@@ -0,0 +1,89 @@
+# Advanced Usage
+
+## Configuration
+
+You can customize some options to meet your needs. To do so, create a `.prettierrc.yml` file at the root of your project with:
+
+```yaml
+# Prettier configuration
+overrides:
+  - files:
+      - '*.java'
+    options:
+      printWidth: 140
+      tabWidth: 4
+      useTabs: false
+      trailingComma: false
+```
+
+Please refer to the [Prettier configuration documentation](https://prettier.io/docs/en/configuration.html) for more information.
+
+## Pre-commit hooks:
+
+To share your Git Hooks, we would suggest to follow this procedure:
+
+1. Create a `package.json` file. For instance:
+   ```bash
+   echo "{}" > package.json
+   ```
+2. Install Husky, Lint-staged, Prettier, the Prettier Java plugin locally `npm install --save-dev husky lint-staged prettier prettier-plugin-java`
+3. Add the following to your `package.json`:
+   ```json
+   "husky": {
```

Comment:
can it be added inside the `package.json` ?
I need to put this inside `.huskyrc` instead

Reply:
You can do both (at least it works for me)